UnitySetup/UnitySetup.psm1

```diff
@@ -212,8 +212,8 @@ function Find-UnitySetupInstaller
 
     foreach($page in $searchPages)
     {
-        $webResult = Invoke-WebRequest $page
-        $prototypeLink = $webResult.Links | Select-Object -ExpandProperty href | Where-Object { 
+        $webResult = Invoke-WebRequest $page -UseBasicParsing
+        $prototypeLink = $webResult.Links | Select-Object -ExpandProperty href -ErrorAction SilentlyContinue | Where-Object { 
```

Comment:
This is required because the new output doesn't always contain an href property, luckily those are links we want to ignore anyways.